### PR TITLE
Ssh add first frames 5734 v1

### PR DESCRIPTION
**Pull request by catenacyber**
Link to ticket: https://redmine.openinfosecfoundation.org/issues/
https://redmine.openinfosecfoundation.org/issues/5734

Describe changes:
- ssh: add frames support (for clear-text records after banner)
- detect: run frames detection on packet disabling app-layer because next packets are encrypted

SV_BRANCH=https://github.com/OISF/suricata-verify/pull/1932

### CODE REVIEW

File: rust/src/ssh/ssh.rs [hunk 1]
```diff
@@ -180,6 +214,30 @@ impl SSHState {
...
+                                &input[SSH_RECORD_HEADER_LEN as usize..],
+                                (head.pkt_len - 2) as i64,
+                                SshFrameType::RecordData as u8,
+                                Some(0),
+                            );
+                            let _pdu = Frame::new(
+                                flow,
+                                stream_slice,
+                                input,
+                                (head.pkt_len + 4) as i64,
```
Comment (catenacyber) [inline rust/src/ssh/ssh.rs:237]: Is this right Victor ?
Setting the frame length that gets over the available data so far...

Comment (victorjulien) [reply rust/src/ssh/ssh.rs:237]: if final length is unknown, it should set -1 and update the frame's len when the final length is available

Comment (catenacyber) [reply rust/src/ssh/ssh.rs:237]: Final length is known but we do not have the full data yet. You just told me it is ok

Comment (victorjulien) [reply rust/src/ssh/ssh.rs:237]: Yup

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 21159

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 21174

**Comment by catenacyber** [comment]
Clean in https://github.com/OISF/suricata/pull/11372

**Pull request closed without merging**